bin/yk-config

```diff
@@ -24,20 +24,20 @@ OUTPUT=""
 usage() {
     echo "Generate C compiler flags for building against the yk JIT.\n"
     echo "Usage:"
-    echo "    yk-config <mode> [--prelink-pipeline <passes>] [--postlink-pipeline <passes>] <--cc|--cxx|--ar|--ranlib|--cppflags|--cflags|--ldflags>\n"
-    echo "    Where <mode> is either 'debug' or 'release'."
+    echo "    yk-config <profile> [--prelink-pipeline <passes>] [--postlink-pipeline <passes>] <--cc|--cxx|--ar|--ranlib|--cppflags|--cflags|--ldflags>\n"
+    echo "    Where <profile> is a Rust cargo profile starting with either 'debug' or 'release'."
```

Comment:
Do we need to update the list of possible values, or does the new profile not use a new subdir in `target`?

Reply:
It creates/uses a new directory in `target`.

Reply:
OK, so should we enumerate the valid profiles?

```
Where <profile> is a Rust cargo profile: 'debug'/'release'/'release-with-debug'."
```